### ISSUE TITLE
Broken first install: cyclic missing; no compatible eco

### ISSUE THREAD

**Issue by garthk** [docpad/docpad-skeleton-bootstrap#28]
First try with a fresh install. 

```
info: Installing the Twitter Bootstrap skeleton into ~/blah 
This can take a moment...
[...]
> js2coffee@0.2.7 preinstall ~/blah/node_modules/docpad/node_modules/cson/node_modules/js2coffee
> npm run-script ./cyclic.js

npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "run-script" "./cyclic.js"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.17

npm ERR! missing script: ./cyclic.js
[...]
> ambi@2.2.0 preinstall ~/blah/node_modules/docpad/node_modules/watchr/node_modules/taskgroup/node_modules/ambi
> node ./cyclic.js

npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.17
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: eco@'>=1.1.0 <1.2.0'
```

`node` v0.10.35, `npm` 2.1.17, `git` 1.9.3, `docpad` v6.69.2.

**Comment by garthk** [comment]
See docpad/docpad#894.

TL;DR: don't use skeletons; docpad team can't or won't keep them up to date.

**Comment by RobLoach** [comment]
@garthk

**Comment by RobLoach** [comment]
> @garthk TL;DR: don't use skeletons; docpad team can't or won't keep them up to date.

Agreed, I've always avoided skeletons and always "rolled my own". Invested some time into https://github.com/RobLoach/generator-docpad to ease the process, but in the end always found it easiest to start from scratch each time. We could push the generator forwards on /docpad/generator-docpad and deprecate the skeletons... Where do you see things going?

**Comment by garthk** [comment]
@balupton [asked](https://github.com/docpad/docpad/issues/894#issuecomment-68859497) for pull requests to mark broken skeletons. I'm not sure there'll be sufficient coverage to fix the out-of-box problem. The future might well resemble the present.

**Comment by claritee** [comment]
Hey guys, any update on this issue?
